sig-and-wg/sig-core/proposals/test-folder-consolidation.md

```diff
@@ -0,0 +1,67 @@
+# Test folder consolidation
+
+## Introduction
+The [tests](https://github.com/kyma-project/kyma/tree/master/tests) folder of the kyma-project contains tests executed against a running Kyma cluster to assure integrity and functional correctness of the cluster with all modules installed.
```

Comment:
```suggestion
The [acceptance tests](https://github.com/kyma-project/kyma/tree/master/tests) folder of the kyma-project contains tests executed against a running Kyma cluster to assure integrity and functional correctness of the cluster with all modules installed.
```
I'm wondering if we maybe can merge the line saying `these are acceptance tests` - in the link name?

Reply:
The folder is called "tests", not acceptance tests

Reply:
Ok - I get it. Let's leave it as it is.